doc/src/pair_hybrid.rst

```diff
@@ -37,36 +42,57 @@ Examples
    pair_coeff * * lj/cut 1.0 1.0
    pair_coeff * * coul/long
 
+   variable one equal ramp(1.0,0.0)
+   variable two equal 1.0-v_one
+   pair_style hybrid/scaled v_one lj/cut 2.5 v_two morse 2.5
+   pair_coeff 1 1 lj/cut 1.0 1.0 2.5
+   pair_coeff 1 1 morse 1.0 1.0 1.0 2.5
+
 Description
 """""""""""
 
-The *hybrid* and *hybrid/overlay* styles enable the use of multiple
-pair styles in one simulation.  With the *hybrid* style, exactly one
-pair style is assigned to each pair of atom types.  With the
-*hybrid/overlay* style, one or more pair styles can be assigned to
-each pair of atom types.  The assignment of pair styles to type pairs
-is made via the :doc:`pair_coeff <pair_coeff>` command.
+The *hybrid*, *hybrid/overlay*, and *hybrid/scaled* styles enable the
+use of multiple pair styles in one simulation.  With the *hybrid* style,
+exactly one pair style is assigned to each pair of atom types.  With the
+*hybrid/overlay* and *hybrid/scaled* styles, one or more pair styles can
+be assigned to each pair of atom types.  The assignment of pair styles
+to type pairs is made via the :doc:`pair_coeff <pair_coeff>` command.
+The *hybrid/scaled* style differs from the *hybrid/overlay* style by
```

Comment:
The only difference between overlay and scaled is that scaled adds a scale factor for each sub-style's contribution ...